thirdeye/thirdeye-frontend/app/mirage/config.js

```diff
@@ -138,4 +138,17 @@ export default function() {
   this.get(`/timeseries/compare/${metric[0].id}/***`, () => {
     return timeseriesCompare;
   });
+
+  /**
+   * Post request for editing alert
+   */
+  this.post(`/thirdeye/entity`, (schema, request) => {
+    let params = request.queryParams && request.queryParams.entityType;
+
+    if (params === 'ANOMALY_FUNCTION') {
+      let requestBody = JSON.parse(request.requestBody);
```

Comment:
any reason we're using `let` instead of `const` here?

Reply:
There's no reason. I'll make the change. Thanks!